### DISCUSSION TITLE
Code working with quasar dev, Already in quasar build, no!

### DISCUSSION THREAD

**Discussion by ewertonrfd** [quasarframework/quasar#14128] [General - Components / Directives / etc] [answered]
Code working with quasar dev
Already in quasar build (production) and then starting server.js does not work
error with map

I ran it in debug mode to see without the minified code
<img width="1841" alt="error duas telas" src="https://user-images.githubusercontent.com/13201575/182513339-971568a3-a35f-4990-b4b5-d52b58cf4fdf.png">

I thought of something related to the environment, for example node
but they are the same

<img width="211" alt="Captura de tela 2022-08-02 235055" src="https://user-images.githubusercontent.com/13201575/182513766-5a4abfe3-83d0-415a-b57e-55f1d196a2ee.png">
<img width="59" alt="Captura de tela 2022-08-02 235123" src="https://user-images.githubusercontent.com/13201575/182513769-dba14c31-5951-4680-9416-18714983c172.png">

If anyone has gone through this, or can help, thanks in advance.

**Comment by pdanpdan** [comment, marked as answer]
Check what are you passing to the `:rows` prop - I suppose you are not passing an array

**Reply by ewertonrfd** [reply]
**You saved me!**
It was the API that made the difference! (although they are the same, the data has changed a little)
Thanks a lot for the array tip, I was tired yesterday 😣

Very, very good
